### ISSUE TITLE
Likes and Notes show in classic editor

### ISSUE THREAD

**Issue by cagrimmett** [janboddez/indieblocks#5]
This looks really cool and I'm looking forward to using it!

I have all of the options enabled, but the Notes and Likes post types are still showing in the Classic editor for me. 

<img width="1076" alt="CleanShot 2022-10-14 at 12 20 27@2x" src="https://user-images.githubusercontent.com/8738771/195894245-3e6ab236-00e7-48cb-bad1-182a7588a67b.png">

Any ideas on how I can get these post types to use the block editor?

**Comment by janboddez** [comment]
Ouch! On my test site (this has actually led me to discover another bug, so thank you!), it works, so I'm assuming a conflict of sorts, perhaps with another plugin.

Here's what **should** be happening: the bit of code that registers the post types checks if the `enable_blocks` setting is _also_ `true`, and, if so, adds `$args['show_in_rest'] = true` to the CPT arguments. See also https://plugins.trac.wordpress.org/browser/indieblocks/tags/0.1.0/includes/class-post-types.php#L139. (Sorry if I get too technical! I'm really hoping we can figure this out, though.)

So, if you have only the CPTs enabled, the classic editor would show, whereas if you have both the block(s) and the CPTs enabled, they oughta automatically use the block editor (and come with a preregistered block template).

Now, the CPTs are definitely there, so that's good. To see whether the "Context" block is available, you could try creating a draft post and look for the `indieblocks/context` block. Or you could look in the database, the `wp_options` table, and search for the `indieblocks_settings` record. Should show a serialized array with an `enable_blocks` boolean in it, which oughta be `1`.

Stupid question, but I'm assuming you've got the block editor working for regular posts? ~~I.e., no Classic Editor plugin, etc.?~~ Scratch that, I just had a look at your site. Maybe a REST API thingy?

**Comment by cagrimmett** [comment]
Thanks for your help @janboddez! 

I could see the Context block in the block editor for my regular posts, and I do not have the Classic Editor plugin installed.

I cloned my site to a staging environment and deactivated all of my plugins except the Indieblocks plugin, which brought back the block editor for Likes and Notes. So turned each plugin back on one-by-one until I found the culprit: [Convert to Blocks by 10up](https://wordpress.org/plugins/convert-to-blocks/).

That plugin isn't critical, it is just nice to have to convert old post to the block editor on the fly. I'm not sure what there is causing issues, but I am leaving it disabled for now so I can use this Indieblocks plugin :)
